src/components/TypeStyles/TypeStyle.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import cn from 'classnames'
+import type {ComponentProps} from 'react'
+
+// minor example
+export function TypeStyle(props: ComponentProps<'typestyle'>) {
```

Comment:
Hey Mark! This seems to be the actual story for Typography, as the classes incorporated into Tailwind config implemented the expected behavior.